eu.ad5001.LogarithmPlotter.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "eu.ad5001.LogarithmPlotter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.DBus.Proprieties",
+        "--talk-name=org.freedesktop.IBus",
+        "--talk-name=org.freedestkop.portal.*",
+        "--filesystem=home"
+    ],
+    "command": "logarithmplotter",
+    "rename-desktop-file": "logarithmplotter.desktop",
+    "rename-icon": "logplotter",
+    "rename-appdata-file": "eu.ad5001.LogarithmPlotter.metainfo.xml",
+    "cleanup": [ ],
+    "post-install": [
+        "install -D mimetypes /app/share/mime/packages/x-logarithm-plot.xml"
+    ],
+    "modules": [
+        "python3-pyside2.json",
```

Comment:
It's preferred to keep all modules a single manifest. You can pretty much copy the PySide2 module from that file here.

Reply:
Oh really? I thought it was good practice to keep shared modules in separate files. Thanks for the advice.

Reply:
the only case is if you have a manifest generated by a tool. Or use "shared-modules".

Reply:
Ah I see

Reply:
Fixed in latest commit.